nkms_eth/project/contracts/PolicyManager.sol

```diff
@@ -200,7 +201,7 @@ contract PolicyManager is Upgradeable {
         uint256 endPeriod = policy.lastPeriod.add(uint(1));
         for (uint256 i = 0; i < policy.nodes.length; i++) {
             address node = policy.nodes[i];
-            if (policy.arrangements[node].disabled) {
+            if (!policy.arrangements[node].active) {
                 continue;
             }
             uint256 nodeRefundValue = revokeArrangement(policy, node, endPeriod);
```

Comment:
Kind of a strange pattern here.  Why not:

```
if (policy.arrangements[node].active) {
    uint256 nodeRefundValue = revokeArrangement(policy, node, endPeriod);
}
```

Reply:
I thought this will be more readable because of small nested "if"